woocommerce/woocommerce
# Remove the first downloadable product note

### All Submissions:






### Changes proposed in this Pull Request:

Remove the first downloadable product note

Part of #34930






### How to test the changes in this Pull Request:



1. Create at least 1 downloadable product
2. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
3. Under Tools -> Cron Events, run `wc_admin_daily`
4. Make sure the note with a title of "Learn more about digital/downloadable products" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
